bin/taste-tester

```diff
@@ -362,28 +362,32 @@ MODES:
     TasteTester::Hooks.get(path)
   end
 
-  case mode.to_sym
-  when :start
-    TasteTester::Commands.start
-  when :stop
-    TasteTester::Commands.stop
-  when :restart
-    TasteTester::Commands.restart
-  when :keeptesting
-    TasteTester::Commands.keeptesting
-  when :status
-    TasteTester::Commands.status
-  when :test
-    TasteTester::Commands.test
-  when :untest
-    TasteTester::Commands.untest
-  when :run
-    TasteTester::Commands.runchef
-  when :upload
-    TasteTester::Commands.upload
-  else
-    logger.error("Invalid mode: #{mode}")
-    puts parser
+  begin
+    case mode.to_sym
+    when :start
+      TasteTester::Commands.start
+    when :stop
+      TasteTester::Commands.stop
+    when :restart
+      TasteTester::Commands.restart
+    when :keeptesting
+      TasteTester::Commands.keeptesting
+    when :status
+      TasteTester::Commands.status
+    when :test
+      TasteTester::Commands.test
+    when :untest
+      TasteTester::Commands.untest
+    when :run
+      TasteTester::Commands.runchef
+    when :upload
+      TasteTester::Commands.upload
+    else
+      logger.error("Invalid mode: #{mode}")
+      puts parser
+      exit(1)
+    end
+  rescue
```

Comment:
errr - that's... bringing the nuclear bomb to the knife fight. Raise a reasonable error and catch it. We shouldn't suppress exceptions that haven't printed nice errors for the user.